extensions/requirements.txt

```
@@ -1,4 +1,5 @@
 ray==2.3.1
 pinecone-client==2.2.1
 llama-cpp-python>=0.1.35
-weaviate-client>=3.16.1
\ No newline at end of file
+weaviate-client>=3.16.1
+deeplake==3.4.1
```

We will be adding bench of new features, better not to fix the version
```suggestion
deeplake>=3.4.1
```